micromanager_gui/main_window.py

```diff
@@ -127,6 +126,24 @@ def __init__(self, viewer: napari.viewer.Viewer, remote=False):
         self.viewer.layers.selection.events.active.connect(self.update_max_min)
         self.viewer.dims.events.current_step.connect(self.update_max_min)
 
+        self._patch_viewer_menu()
+
+    def _patch_viewer_menu(self):
+        self._menu = QtW.QMenu("&Micro-Manager")
+
+        action = self._menu.addAction("Device Property Browser...")
+        action.triggered.connect(self._show_prop_browser)
+
+        v = getattr(self.viewer, "__wrapped__", self.viewer)
+        bar = v.window._qt_window.menuBar()
+        bar.insertMenu(list(bar.actions())[-1], self._menu)
```

Comment:
I've wondered how to do this before. maybe a good feature request for napari is a method to add these without accessing private variables like that (or know about wrapped)

Reply:
Can we pick up the napari theming so that the menu doesn't look out of place?

![menu-themeing](https://user-images.githubusercontent.com/10111092/157097340-acbdd1da-d996-4e2e-b2c0-22c2f77e6700.gif)

Reply:
> maybe a good feature request for napari is a method to add these without accessing private variables like that (or know about wrapped)

it's [coming in npe2](https://github.com/napari/npe2/blob/main/npe2/manifest/menus.py).  don't tell anyone about this workaround :joy:

Reply:
sure, can fix the theme